lib/src/admob_banner.dart

```diff
@@ -11,13 +11,15 @@ class AdmobBanner extends StatefulWidget {
   final AdmobBannerSize adSize;
   final void Function(AdmobAdEvent, Map<String, dynamic>) listener;
   final void Function(AdmobBannerController) onBannerCreated;
+  final bool nonPersonalizedAds;
```

Comment:
Not a deal-breaker but would be nice to have more docs on this, and a link. https://support.google.com/admob/answer/7676680

Reply:
Also adding to the README the use of [admob_consent](https://pub.dev/packages/admob_consent) as a nice way to get user's consent.

Reply:
I'm gonna do that !